testing/unit/test_configurator.py

```diff
@@ -0,0 +1,47 @@
+"""Unit tests for configurator"""
+
+import unittest
+import os
+
+from daq.configurator import Configurator, print_config
+
+def dict_to_str(obj):
+    """Dump dict as key=value to str object."""
+    config_list = []
+    for key in sorted(obj.keys()):
+        value = obj[key]
+        quote = '"' if ' ' in str(value) else ''
+        config_list.append("%s=%s" % (key, obj[key]))
+    return '\n'.join(config_list)
+
+
+class TestConfigurator(unittest.TestCase):
+    """Test class for Configurator"""
+
+    config = {
+        'monitor_scan_sec' : '30',
+        'default_timeout_sec' : '350',
+        'base_conf' : 'misc/module_config.json',
+        'site_path' : 'local/site/',
+        'initial_dhcp_lease_time' : '120s',
+        'dhcp_lease_time' : '500s',
+        'long_dhcp_response_sec' : '105'
+    }
+
+    def setUp(self):
+        tmpfile = open('temp.conf', 'w+')
+        tmpfile.write(dict_to_str(self.config))
+        tmpfile.close()
+
+    def tearDown(self):
+        os.remove('temp.conf')
+
+    def test_config_load(self):
+        """Test config is loaded properly"""
+        configurator = Configurator()
+        args = ['test', 'temp.conf']
+        read_config = configurator.parse_args(args)
```

Comment:
I hate English: red == read != read

Reply:
I am not sure I understand, Could you elaborate.

Reply:
"Read" past tense is different than "read" present tense.  I was going to make a comment about how you should use "read" instead of "read"... and then I hate english.